api/app/Models/Category.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Category extends Model
+{
+    use HasFactory;
+    
+    protected $fillable = [
+        'label',
+    ];
+    public function stores()
```

Comment:
- add space 
- i think it should be orders instead of stores

```suggestion
    ];
    
    public function orders()
```